animations-baker.js

```diff
@@ -330,9 +329,11 @@ const baker = async (uriPath = '', fbxFileNames, vpdFileNames, outFile) => {
       const rootBone = object; // not really a bone
       const leftFootBone = bones.find(b => b.name === 'mixamorigLeftFoot');
       const rightFootBone = bones.find(b => b.name === 'mixamorigRightFoot');
+      const allOnes = arr => arr[0] === 1 && arr.every(v => v === arr[0]);
```

Comment:
What a weird check. Why is the first item special? If you want to know there is at least one item, checking `.length` is better.

Reply:
removed redundant check